src/overlays/actors/ovl_En_Zo/z_en_zo.c

```diff
@@ -336,9 +336,9 @@ static Gfx sTransparencyDlist[] = {
 void EnZo_Draw(Actor* thisx, GlobalContext* globalCtx) {
     EnZo* this = THIS;
     s32 i;
-    u8* shadowTex = GRAPH_ALLOC(globalCtx->state.gfxCtx, sizeof(u8) * SQ(64));
+    u8* shadowTex = GRAPH_ALLOC(globalCtx->state.gfxCtx, (s32)sizeof(u8[64][64]));
```

Comment:
The cast is required for the `for` loop down below. I wouldn't be surprised if they had just used just `4096` or `64 * 64` directly, instead of with `sizeof`.

Reply:
If it's only used internally I'd put it in the C file.

Reply:
Its not internal, its used by the actors that make use of the dynamic shadow. Such as `EnZo` here.

Reply:
Then yeah, making a define definitely makes sense.